packages/tooltips/src/styled/StyledLightTooltip.ts

```diff
@@ -5,27 +5,62 @@
  * found at http://www.apache.org/licenses/LICENSE-2.0.
  */
 
-import styled from 'styled-components';
-import classNames from 'classnames';
-import TooltipStyles from '@zendeskgarden/css-tooltips';
-import { retrieveComponentStyles } from '@zendeskgarden/react-theming';
+import styled, { DefaultTheme, css, ThemeProps } from 'styled-components';
+import { retrieveComponentStyles, getColor, DEFAULT_THEME } from '@zendeskgarden/react-theming';
+import stripUnit from 'polished/lib/helpers/stripUnit';
 
 import { StyledTooltip, IStyledTooltipProps } from './StyledTooltip';
+import { StyledTitle } from './StyledTitle';
 
 const COMPONENT_ID = 'tooltip.light_tooltip';
 
+const arrowStyles = ({ theme, size }: IStyledTooltipProps & ThemeProps<DefaultTheme>) => {
+  let arrowFontSize = `${theme.space.base * 1.75 + 2}px`;
+
+  if (size === 'large') {
+    arrowFontSize = `${stripUnit(theme.fontSizes.sm) + 2}px`;
+  } else if (size === 'extra-large') {
+    arrowFontSize = `${theme.space.base * 4 + 2}px`;
+  }
+
+  return css`
+    &::before,
+    &::after {
+      font-size: ${arrowFontSize};
+    }
+  `;
+};
```

Comment:
Incorporating this into the `StyledTooltip` code should make this slightly less obtuse. It's always just whatever the `small`/`medium`/`large` sizes are plus `2px` (basically accounting for two `borderWidths.sm` strokes).

Reply:
I have moved this to the `StyledTooltip` component